File: web-front/src/main/java/com/isystk/sample/web/front/controller/api/v1/post/PostRestForm.java
package com.isystk.sample.web.front.controller.api.v1.post;

import java.io.Serializable;

public class PostRestForm implements Serializable {

  private static final long serialVersionUID = 7593564324192730932L;

  String title;

  String text;

  public String getTitle() {
    return this.title;
  }

  public String getText() {
    return this.text;
  }

  public void setTitle(String title) {
    this.title = title;
  }

  public void setText(String text) {
    this.text = text;
  }
}

File: web-front/src/main/kotlin/com/isystk/sample/web/front/controller/api/v1/post/PostRestForm.kt
package com.isystk.sample.web.front.controller.api.v1.post

import lombok.Getter
import lombok.Setter
import java.io.Serializable

@Setter
@Getter
class PostRestForm : Serializable {
    var title: String? = null
    var text: String? = null

    companion object {
        private const val serialVersionUID = 7593564324192730932L
    }
}